entropy/entropy.go

```diff
@@ -0,0 +1,39 @@
+package entropy
+
+import (
+	"crypto/rand"
+	"fmt"
+)
+
+func GetRandom(len uint32) ([]byte, error) {
```

Comment:
This feature is intended to be pretty generic. Hence, I'd like that it should be easy to change / swap different randomness sources. With the current format, one has to (un-)comment lines of code. A more "ergonomic" way as you say, would be to use interfaces ;) Since the functionality required is similar to the Go `io.Reader` interface, we can just re-use that. That way, we can simply pass in as an argument the entropy source we would like. Something along the lines of:
```go
func GetRandom(source io.Reader, len uint32) {
   if source == nil {
     source = rand.Reader
   }
   ...
}
```

In the same package, we would then be able to implement all the source we want and pick them easily with `GetRandom(entropy.LavaLamp,32)` or `GetRandom(entropy.NISTBeacon,32)`. Eventually, this will be a field in the config file but that can wait ;)

Reply:
> Eventually, this will be a field in the config file but that can wait ;)
Actually, *if* we want to use different sources of randomness for the launch, we might want to make this configurable from the beginning, or in a very soon-to-be PR. What do you think ?
The work is basically to add a field to the config, add an option to the CLI and link the two.

Reply:
I wonder if we even need to add this as a flag in the cli--it looks like adding one's own randomness source involves implementing an interface with a `Read` function, which means that enabling custom randomness from the command line/config file is unnecessary if it's already a change that requires customizing drand code. What do you think? Is there a straightforward way to implementing custom randomness without making changes to the entropy class introduced here?

Reply:
Yes, it may be too complex for now: let's keep it simple for now and have stg running first ;)